src/integrator/DPDThermostat.cpp

```diff
@@ -142,46 +172,96 @@ void DPDThermostat::frictionThermoTDPD(Particle& p1, Particle& p2)
     // Implements a transverse DPD thermostat with the canonical functional form of omega
     Real3D r = p1.position() - p2.position();
     real dist2 = r.sqr();
+    System& system = getSystemRef();
 
-    if (dist2 < current_cutoff_sqr)
+    if (system.shearRate == .0)
```

Comment:
== floating point comparisons are flaky. Is there another way?

Reply:
Changed to boolean